lib/msal-browser/src/app/Configuration.ts

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// import { Logger } from "./Logger";
+import { BrowserUtils } from "../utils/BrowserUtils";
+import { AuthOptions } from "msal-common";
+// import { TelemetryEmitter } from "./telemetry/TelemetryTypes";
+
+/**
+ * Cache location options supported by MSAL are:
+ * - local storage: MSAL uses browsers local storage to store its cache
+ * - session storage: MSAL uses the browsers session storage to store its cache
+ */
+export type CacheLocation = "localStorage" | "sessionStorage";
```

Comment:
We should change this. This is causing issues in implicit-flow with type widening in certain frameworks.

Reply:
Can you explain the issue or provide an alternative?

Reply:
Make an enum in Constants.ts.